app/helpers/users.js

```diff
@@ -14,6 +14,22 @@ exports.encryptPass = pass => {
   }
 };
 
-exports.generateToken = user => jwt.sign(user, config.common.secretPass, { expiresIn: '2h' });
+exports.generateToken = user =>
+  jwt.sign(user, config.common.secretPass, { expiresIn: config.common.expiresIn });
 
 exports.comparePass = (pass, passEncrypted) => bcrypt.compare(pass, passEncrypted);
+
+exports.getPagination = (page, size) => {
+  const limit = size || config.common.database.limit;
+  const offset = page ? page * limit : config.common.database.offset;
+  return { limit, offset };
+};
+
+exports.getPaginData = (data, page, limit) => {
```

Comment:
Use this convention for a paginated response: https://github.com/Wolox/tech-guides/blob/master/backend/backend-standard-and-best-practices.md#31--pagination